hera_qm/tests/test_utils.py

```diff
@@ -121,8 +121,8 @@ def test_get_metrics_ArgumentParser_omnical_metrics():
     nt.assert_equal(args.extension, 'foo')
 
 
-def test_get_metrics_ArgumentParser_xrfi_run():
-    a = utils.get_metrics_ArgumentParser('xrfi_run')
+def test_get_metrics_ArgumentParser_xrfi_h1c_run():
+    a = utils.get_metrics_ArgumentParser('xrfi_h1c_run')
```

Comment:
The old `xrfi_run` is now labeled `xrfi_h1c_run`. The general idea is that current strategies get the privilege of simple names, and legacy ones get labels indicating which analysis they were used for.